core/src/main/java/org/kiwix/kiwixmobile/core/di/components/CoreComponent.kt

```diff
@@ -105,4 +106,5 @@ interface CoreComponent {
   fun inject(searchActivity: SearchActivity)
   fun inject(helpActivity: HelpActivity)
   fun inject(settingsActivity: CoreSettingsActivity)
+  fun inject(fileLogger: FileLogger)
```

Comment:
this method can be deleted, in Components you declare injection sites for field Injection ie Activities/Fragments/Services, classes that we cannot control the construction of but want to be able to inject objects into